TheMovieDb/Controllers/DetailViewController.swift

```diff
@@ -9,34 +9,194 @@ import UIKit
+        view.translatesAutoresizingMaskIntoConstraints = false
+        view.font = UIFont.boldSystemFont(ofSize: 18)
+        view.textColor = .label
+        view.setContentHuggingPriority(UILayoutPriority(1), for: .horizontal)
+        return view
+    }()
+    
+    lazy private var mediaAndReleaseStackView: UIStackView = {
+        let view = UIStackView()
+        view.translatesAutoresizingMaskIntoConstraints = false
+        view.axis = .horizontal
+        view.spacing = 5
+        view.distribution = .fill
+        view.alignment = .fill
+        view.setContentHuggingPriority(UILayoutPriority(1), for: .vertical)
+        view.setContentHuggingPriority(UILayoutPriority(1), for: .horizontal)
+        
+        view.addArrangedSubview(movieMediaType)
+        view.addArrangedSubview(movieReleaseDate)
+        return view
+    }()
+    
+    lazy private var starImageView: UIImageView = {
+        let view = UIImageView()
+        view.image = UIImage(systemName: "star.fill")
+        view.tintColor = .systemYellow
+        view.translatesAutoresizingMaskIntoConstraints = false
+        view.contentMode = .scaleAspectFit
+        return view
+    }()
+    
+    lazy private var ratingStackView: UIStackView = {
+        let view = UIStackView()
+        view.translatesAutoresizingMaskIntoConstraints = false
+        view.axis = .horizontal
+        view.spacing = 5
+        view.distribution = .fill
+        view.alignment = .fill
+        view.setContentHuggingPriority(UILayoutPriority(252), for: .vertical)
+        view.setContentCompressionResistancePriority(UILayoutPriority(751), for: .horizontal)
+        
+        view.addArrangedSubview(starImageView)
+        view.addArrangedSubview(movieRating)
+        return view
+    }()
+    
+    lazy private var scrollView: UIScrollView = {
+        let view = UIScrollView()
+        view.translatesAutoresizingMaskIntoConstraints = false
+        
+        view.addSubview(moviePoster)
+        view.addSubview(movieTitle)
+        view.addSubview(mediaAndReleaseStackView)
+        view.addSubview(movieOverview)
+        view.addSubview(ratingStackView)
+        
+        return view
+    }()
+    
+    lazy private var closeButton: UIButton = {
+        let view = UIButton(type: UIButton.ButtonType.close)
+        view.translatesAutoresizingMaskIntoConstraints = false
+        if #available(iOS 15.0, *) {
+            view.configuration = UIButton.Configuration.filled()
+        }
+        view.tintColor = .systemIndigo
+        view.addTarget(self, action: #selector(closeView), for: .touchUpInside)
+        return view
+    }()
 
-    var configurationImages: ConfigurationImages?
+    var configurationImage: ConfigurationImage?
     var movieItem: MovieItem?
     
+    // General margin for ui elements
+    private let margin: CGFloat = 10
+    
+    // General corner radius
+    private let cornerRadius: CGFloat = 10.0
+    
     static let segueIdentifier = "goToMovieDetalSegue"
```

Comment:
If we got rid of the Storyboard, maybe we don't need this identifier, right?

Reply:
Thanks, I'll remove it